templates/security-proxy/security-proxy-deployment.yaml

```diff
@@ -50,6 +50,27 @@ spec:
           {{- $webapp.extra_environment | toYaml | nindent 10 }}
           {{- end }}
       {{- end }}
+      - name: prepare-proxy-webapp
+        image: {{ $webapp.docker_image }}
+        command:
+          - /bin/sh
+          - -c
+          - cp -r /var/lib/jetty/webapps/ROOT/* /mnt/security_proxy_webapp ;
+            if [ -d /mnt/georchestra_datadir/security-proxy/ressources/ ] ; then
+              cp /mnt/georchestra_datadir/security-proxy/ressources/* /mnt/security_proxy_webapp/ ;
```

Comment:
--force by default ?

Reply:
without it seems working, as here : https://github.com/georchestra/helm-georchestra/pull/52/files#diff-b058540d36c74466303361ec6dd03c7d0adf765a2bdc339a46dde277182ec376R51

Reply:
looks like, the other cp being made on other places don't seem to require the option.